.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    env:
+      HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
```

Comment:
@romarickb We have to whitelist the secret to be injected into this repo. Are you ready to use it?